content/en/docs/install-operate/running/index.md

```diff
@@ -272,6 +272,7 @@ docker run --rm -i -t \
            --privileged \
            -v /var/run/docker.sock:/host/var/run/docker.sock \
            -v /proc:/host/proc:ro \
+           -v /etc:/host/etc:ro \
            falcosecurity/falco-no-driver:latest falco --modern-bpf
 ```
 
```

Comment:
* `-v /var/run/docker.sock:/host/var/run/docker.sock`  needed to scrape docker container info and extract fields like `container_name`. Please note that if the container is spawned with another container runtime like `containerd` you need to mount the corresponding socket `-v /var/run/containerd/containerd.sock:/host/var/run/containerd/containerd.sock` and provide Falco with the corresponding command line flag `--cri /var/run/containerd/containerd.sock`
* `-v /proc:/host/proc:ro` needed to read the list of processes running on the host.
* `-v /etc:/host/etc:ro` needed to resolve users on the host.